src/ImageDimensionsInterface.php

```diff
@@ -16,11 +16,25 @@
 
 interface ImageDimensionsInterface
 {
+    public const ORIENTATION_NORMAL = 1;
+    public const ORIENTATION_NORMAL_90 = 6;
+    public const ORIENTATION_NORMAL_180 = 3;
+    public const ORIENTATION_NORMAL_270 = 8;
+    public const ORIENTATION_MIRROR = 2;
+    public const ORIENTATION_MIRROR_90 = 7;
+    public const ORIENTATION_MIRROR_180 = 4;
```

Comment:
Are these the only valid orientations or just the ones you want to support? Is there an exif standard? If so, you might want to add a comment that they relate :)

Reply:
These are all orientations from the standard, I added a comment in 092c5ba58219714c928a854567bd6c2fe5d64827.

Do you find the naming of the constants intuitive?  
Or would the following be better?

```php
public const ORIENTATION_NORMAL = 1;
public const ORIENTATION_90 = 6;
public const ORIENTATION_180 = 3;
public const ORIENTATION_270 = 8;
public const ORIENTATION_MIRRORED = 2;
public const ORIENTATION_90_MIRRORED = 7;
public const ORIENTATION_180_MIRRORED = 4;
public const ORIENTATION_270_MIRRORED = 5;
```

Reply:
b6468db7b47790c6bb60938b1a7e24e1ada90ec9

Reply:
What is the meaning of the constant `ORIENTATION_MIRROR_90`? Is the image mirrored or is the orientation counter clockwise?

Reply:
`ORIENTATION_MIRROR_90` means mirror the image and then rotate it 90 degrees clockwise

Reply:
Ok, I understand. Didn't know that there is a mirror tag in the Exif standard. But makes sense for the portrait cameras in smartphones.